perses/analysis/load_simulations.py

```diff
@@ -0,0 +1,303 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import os
+import itertools
+import sys
+from perses.analysis import utils
+from openeye import oechem, oegraphsim
+#from perses.utils import openeye
+from openmoltools.openeye import smiles_to_oemol, generate_conformers
+import logging
+
+
+from pymbar import timeseries
+from pymbar import MBAR
+
+_logger = logging.getLogger("analysis")
+
+
+class molecule(object):
+    def __init__(self, i, string):
+        self.line = string
+        details = string.split(';')
+        self.index = i
+        self.smiles, self.name, self.exp, self.experr, self.calc, self.calcerr = details[1:7]
+        self.mol = openeye.smiles_to_oemol(self.smiles)
+        self.mol = generate_conformers(self.mol,max_confs=1)
+        self.exp = kcal_to_kt(float(self.exp))
+        self.experr = kcal_to_kt(float(self.experr))
+        self.calc = kcal_to_kt(float(self.calc))
+        self.calcerr = kcal_to_kt(float(self.calcerr))
+        self.mw = self.calc_mw()
+        self.ha = self.heavy_atom_count()
+
+    def calc_mw(self):
```

Comment:
When it doesn't create huge hardship, we should prefer full words, like `compute_molecular_weight`, to `calc_mw`, which are harder to process.

We also, in general, always want to write docstrings, ideally conforming to the [numpy docstring convention](https://numpydoc.readthedocs.io/en/latest/format.html).